test/e2e/common/configmap_volume.go

```diff
@@ -184,7 +184,12 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 		Eventually(pollLogs, podLogTimeout, framework.Poll).Should(ContainSubstring("value-2"))
 	})
 
-	It("binary data should be reflected in volume [NodeConformance]", func() {
+	/*
+		Release: v1.12
+		Testname: configmap-supports-binary-data
+		Description: Make sure config map data and binary data are readable on volumes mounted in containers.
```

Comment:
There should be "MUST" statements in here to explain what this test is actually checking.  See other tests in here for examples.

Reply:
Updated comments. 
Thanks for pointing. Different e2e test files are following different semantics/format (hyphen separated testname or k8s-component: feature1,feature2) to express Testname and Description. Does it make sense to follow the same format wrt. to other e2e test's description and name belonging the same file or are we drafting any guideline to follow same structure for every conformance tests. I have gone through [existing guidelines](https://github.com/kubernetes/community/blob/3ee8007c329b0c726d803d35ab3e0bec2eaf2191/contributors/devel/conformance-tests.md#test-metadata)